runtime/compiler/env/VMJ9.h

```diff
@@ -331,6 +331,7 @@ class TR_J9VMBase : public TR_FrontEnd
    virtual bool supportsGuardMerging()        { return true; }
    virtual bool canDevirtualizeDispatch()     { return true; }
    virtual bool doStringPeepholing()          { return true; }
+   virtual bool isPortableSCCEnabled()        { return false; }
```

Comment:
I would add a `TR_ASSERT_FATAL` here. The notion of Portable SCC is independent of the compiler but from a conceptual integrity point of view it doesn't make sense for the compiler to ask that question without a `TR_J9SharedCacheVM` in hand, so rather than silently returning false, we should add an assert so that no one asks this question with the wrong Front End object.